api/v1alpha1/podhttpchaos_types.go

```diff
@@ -181,6 +186,26 @@ const (
 	PodHttpResponse PodHttpChaosTarget = "Response"
 )
 
+// PodHttpChaosTLS contains the tls config for HTTPChaos
+type PodHttpChaosTLS struct {
+	// SecretName represents the name of required secret resource
+	SecretName string `json:"secretName,omitempty"`
+
+	// SecretNamespace represents the namespace of required secret resource,
+	// should be the same with deployment/chaos-controller-manager in most cases
+	SecretNamespace string `json:"secretNamespace,omitempty"`
+
+	// CertName represents the data name of cert file in secret, `tls.crt` for example
+	CertName string `json:"certName,omitempty"`
+
+	// KeyName represents the data name of key file in secret, `tls.key` for example
+	KeyName string `json:"KeyName,omitempty"`
+
+	// CAName represents the data name of ca file in secret, `ca.crt` for example
+	// +optional
+	CAName *string `json:"caName,omitempty"`
+}
```

Comment:
- if these fields are required, please remove the `omitempty` tag and append `+kubebuilder:validation:Required`
- if they are optional, please add `+optional` comment

Reply:
Fixed.